packages/ui/src/views/repo/repo-branch-rules/components/repo-branch-rules-fields.tsx

```diff
@@ -194,10 +192,13 @@ export const BranchSettingsRuleTargetPatternsField: FC<FieldProps> = ({ setValue
 const BranchSettingsRuleBypassListOption = (option: MultiSelectOptionType<BypassUsersList>) => {
   return (
     <>
-      <Avatar className="overflow-hidden rounded-md" size="6">
-        {!!option.url && <AvatarImage src={option.url} alt={option.display_name} />}
-        <AvatarFallback>{getInitials(option.display_name)}</AvatarFallback>
-      </Avatar>
+      <Avatar.Root>
+        {option.url ? (
+          <Avatar.Image src={option.url} alt={option.display_name} />
+        ) : (
+          <Avatar.Fallback>{getInitials(option.display_name)}</Avatar.Fallback>
+        )}
```

Comment:
Same here, `Fallback` shouldn't be conditional.
```suggestion
        {option.url && <Avatar.Image src={option.url} alt={option.display_name} />}
        <Avatar.Fallback>{getInitials(option.display_name)}</Avatar.Fallback>
```